salsa_dancing_molecules/worker_process/infiles_handler.py

```diff
@@ -0,0 +1,20 @@
+"""Program for loading atoms object from a file."""
+
+import json
+from pickle import load
+
+
+def handle_files(file_path):
+    """
+    Read a specified .json file and extracts atoms object.
+
+    Args:
+        file_path - The file path to the .json file containing
+                    information about the atoms object.
```

Comment:
To me it is not perfectly clear what "simulation_info" is. Since this is returned by the function it may be good to explain it in the function description.